test-utils/state-viewer/src/cli.rs

```diff
@@ -83,6 +85,9 @@ pub enum StateViewerSubCommand {
     /// Print `EpochInfo` of an epoch given by `--epoch_id` or by `--epoch_height`.
     #[clap(name = "epoch_info")]
     EpochInfo(EpochInfoCmd),
+    /// Dump stats for the RocksDB storage.
+    #[clap(name = "rocksdb_stats")]
```

Comment:
unrelated to PR, but I am surprised that we need all these `name = ` attributes -- feels like something clap should just infer from the enum names...